composer.json

```diff
@@ -33,24 +33,29 @@
         "doctrine/common": "^3",
         "doctrine/doctrine-bundle": "^2.2",
         "doctrine/orm": "^2.7",
-        "endroid/qr-code-bundle": "^3.3",
+        "endroid/qr-code": "^4",
         "ezyang/htmlpurifier": "^4.6",
         "graze/dog-statsd": "^1",
         "knplabs/knp-menu-bundle": "^3",
         "knpuniversity/oauth2-client-bundle": "^2.8",
         "laminas/laminas-diagnostics": "^1.6",
         "laminas/laminas-feed": "^2.13",
-        "league/oauth2-github": "^2.0",
+        "league/oauth2-github": "^3.0",
         "nelmio/cors-bundle": "^2",
         "nelmio/security-bundle": "^2.4",
-        "pagerfanta/pagerfanta": "^2.4",
+        "pagerfanta/doctrine-orm-adapter": "^3.1",
+        "pagerfanta/pagerfanta": "^3.1",
```

Comment:
If you're installing the `pagerfanta/pagerfanta` monorepo, you don't need the sub-split packages (ORM adapter and Twig) since those are replaced.

Reply:
that requirement should probbaly be on `pagerfanta/core` instead

Reply:
Thanks, fixed